input/ignoreWarnings.txt

```diff
@@ -636,3 +636,276 @@ INFORMATION: QuestionnaireResponse/QuestionnaireResponse-patients-fetal-death-ca
+INFORMATION: Bundle/bundle-jurisdiction-live-birth-babyg-quinn: Bundle.entry[30].resource.category[0]: Reference to draft CodeSystem http://terminology.hl7.org/CodeSystem/condition-category|0.5.0
+INFORMATION: Bundle/bundle-jurisdiction-live-birth-babyg-quinn: Bundle.entry[36].resource.category[0]: Reference to draft CodeSystem http://terminology.hl7.org/CodeSystem/condition-category|0.5.0
+INFORMATION: Bundle/bundle-jurisdiction-live-birth-babyg-quinn: Bundle.entry[37].resource.category[0]: Reference to draft CodeSystem http://terminology.hl7.org/CodeSystem/condition-category|0.5.0
+INFORMATION: Bundle/bundle-jurisdiction-live-birth-babyg-quinn: Bundle.entry[57].resource.category[0]: Reference to draft CodeSystem http://terminology.hl7.org/CodeSystem/condition-category|0.5.0
+INFORMATION: Bundle/bundle-jurisdiction-live-birth-babyg-quinn: Bundle.entry[58].resource.category[0]: Reference to draft CodeSystem http://terminology.hl7.org/CodeSystem/condition-category|0.5.0
+INFORMATION: Bundle/bundle-provider-live-birth-babyg-quinn: Bundle.entry[2].resource/*Encounter/encounter-birth-babyg-quinn*/.hospitalization.dischargeDisposition: Reference to draft CodeSystem http://terminology.hl7.org/CodeSystem/discharge-disposition|0.1.0
+INFORMATION: Bundle/bundle-provider-live-birth-babyg-quinn: Bundle.entry[0].resource.entry[2].resource/*Encounter/encounter-birth-babyg-quinn*/.hospitalization.dischargeDisposition: Reference to draft CodeSystem http://terminology.hl7.org/CodeSystem/discharge-disposition|0.1.0
+INFORMATION: Bundle/bundle-provider-live-birth-babyg-quinn: Bundle.entry[24].resource.category[0]: Reference to draft CodeSystem http://terminology.hl7.org/CodeSystem/condition-category|0.5.0
+INFORMATION: Bundle/bundle-provider-live-birth-babyg-quinn: Bundle.entry[25].resource.category[0]: Reference to draft CodeSystem http://terminology.hl7.org/CodeSystem/condition-category|0.5.0
+INFORMATION: Bundle/bundle-provider-live-birth-babyg-quinn: Bundle.entry[27].resource.category[0]: Reference to draft CodeSystem http://terminology.hl7.org/CodeSystem/condition-category|0.5.0
+INFORMATION: Bundle/bundle-provider-live-birth-babyg-quinn: Bundle.entry[29].resource.category[0]: Reference to draft CodeSystem http://terminology.hl7.org/CodeSystem/condition-category|0.5.0
+INFORMATION: Bundle/bundle-provider-live-birth-babyg-quinn: Bundle.entry[33].resource.category[0]: Reference to draft CodeSystem http://terminology.hl7.org/CodeSystem/condition-category|0.5.0
+INFORMATION: Bundle/bundle-provider-live-birth-babyg-quinn: Bundle.entry[39].resource.category[0]: Reference to draft CodeSystem http://terminology.hl7.org/CodeSystem/condition-category|0.5.0
+INFORMATION: Bundle/bundle-provider-live-birth-babyg-quinn: Bundle.entry[40].resource.category[0]: Reference to draft CodeSystem http://terminology.hl7.org/CodeSystem/condition-category|0.5.0
```

Comment:
Is there a way we could reasonably specify version 5.0.0 rather than it looking for 5.5.0 and giving this draft warning? I don't see what the difference between them is, maybe I just don't understand codesystems fully.
[5.5.0](https://terminology.hl7.org/5.5.0/CodeSystem-condition-category.html)
[5.0.0](https://terminology.hl7.org/5.0.0/CodeSystem-condition-category.html)

Reply:
This is beyond our control.   HL7 Terminology folks need to get their $#*@ together.